setup.cfg

```
@@ -4,7 +4,7 @@ description = A sunpy FIDO plugin for accessing data in the Solar Orbiter Archiv
 long_description = file: README.rst
 author = David Stansby and The SunPy Community
 author_email = sunpy@googlegroups.com
-url =
+url = https://sunpy.org
```

```suggestion
url = https://github.com/sunpy/sunpy-soar
```
or maybe the docs is better?